src/Splat.Adapters.Tests/SimpleInjectorDependencyResolverTests.cs

```diff
@@ -9,47 +9,47 @@
 using Splat.SimpleInjector;
 using Xunit;
 
-namespace Splat.Simplnjector
+namespace Splat.Adapters.Tests
 {
     /// <summary>
     /// Tests to show the <see cref="SimpleInjectorDependencyResolver"/> works correctly.
     /// </summary>
-    public class DependencyResolverTests
+    public class SimpleInjectorDependencyResolverTests
     {
         /// <summary>
-        /// Simples the injector dependency resolver should resolve a view model.
+        /// SimpleInjectorDependencyResolver should resolve a view model.
         /// </summary>
         [Fact]
         public void SimpleInjectorDependencyResolver_Should_Resolve_View_Model()
         {
             var container = new Container();
-            container.Register<ViewModel>();
+            container.Register<ViewModelOne>();
             container.UseSimpleInjectorDependencyResolver();
 
-            var viewModel = Locator.Current.GetService(typeof(ViewModel));
+            var viewModel = Locator.Current.GetService(typeof(ViewModelOne));
```

Comment:
Be careful with using the static Locator. 

You should reset it first by doing 

Locator.CurrentMutable = new Modern...();

Since it could of been changed by previous test  that have been run in this scenario.

Reply:
I attempted this and still ran into issues.  I punted to re organizing into solution folders.  This way each static Locator can execute in it's own app domain without collision.